krun/platform.py

```diff
@@ -26,13 +26,11 @@
 class BasePlatform(object):
     __metaclass__ = ABCMeta
 
-    # Each temperature sensor must cool to its starting temperature plus
-    # TEMP_THRESHOLD_DEGREES. An absolute value in degrees centigrade.
-    TEMP_THRESHOLD_DEGREES = 1
+    TEMP_THRESHOLD_DEGREES = 3
 
     TEMP_MANDATORY_WAIT = 1
     TEMP_POLL_FREQ = 10                 # seconds between polls
-    TEMP_POLLS_BEFORE_MELTDOWN = 60     # times 10 = ten mins
+    TEMP_POLLS_BEFORE_MELTDOWN = 360    # times 10 = 1 hour
```

Comment:
I don't think that matters too much either way. That said, we're probably more likely to hit our threshold if we poll more often, so perhaps 1s is better...